src/generators/legacy-html/assets/style.css

```diff
@@ -591,38 +599,26 @@ hr {
 .toc ul {
   margin: 0;
 }
-
-.toc li a::before {
-  content: '■';
-  color: var(--color-text-primary);
-  padding-right: 1em;
-  font-size: 0.9em;
+.toc > ul:first-child {
```

Comment:
We’ve been adding empty lines between css declaration blocks on this repo.

would you mind doing the same here? It seems that your commits reverted me adding empty lines 🙈